### PR TITLE
Feature/tg 70/list task page

### PR DESCRIPTION
**Pull request by mpueblas**
Tg 70: list task page - Martín Pueblas

### CODE REVIEW

File: src/Components/Tasks/List/Row/index.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      <td>{parentProject ? parentProject.name : 'No parent project'}</td>
+      <td>{taskCreatorId}</td>
+      <td>{taskName}</td>
+      <td>{taskDescription}</td>
+      <td>
+        {assignedEmployee && assignedEmployee.length
+          ? assignedEmployee.map((employee) => employee.firstName)
+          : 'no tiene id'}
+      </td>
+      <td>{startDate}</td>
```
Comment (martinalbrnz) [inline src/Components/Tasks/List/Row/index.jsx:27]: lo ideal seria que lo formatees a regional date con algo asi
`<td>{new Date(startDate).toLocaleDateString()}</td>`

**Review by JuanBSantomero — APPROVED** [review]
Everything looks great!

File: src/Components/Tasks/List/Row/index.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <td>{status}</td>
+      <td>
+        <i className="fa-solid fa-xmark" onClick={() => props.deleteTask(id)}></i>
+      </td>
+      <td>
+        <a href={`http://localhost:3000/tasks-edit/${id}`}>
+          <i className="fa-solid fa-pen-to-square"></i>
+        </a>
+      </td>
```
Comment (martinalbrnz) [inline src/Components/Tasks/List/Row/index.jsx:36]: Si se visualiza mal, aca dberias agregar dos <td> mas para los espacios de los botones de editar y eliminar. Si no, no es necesario.

File: src/Components/Tasks/List/index.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+function List() {
+  const [tasks, setTask] = useState([]);
+  useEffect(() => {
+    fetch('http://localhost:8080/tasks')
+      .then((response) => response.json())
+      .then((response) => {
+        setTask(response.data);
+        console.log(response.data);
+      });
```
[the remark this answers is not in the document]
Comment (mpueblas) [reply src/Components/Tasks/List/index.jsx]: fixed

**Review by martinalbrnz — APPROVED** [review]
LGTM!

**Comment by RominaRIP** [comment]
## Testing Results

Martín, please check the functionality of the table list.

Tested locally on https://alfonso-trackgenix-app-git-feature-tg-70listtaskpage-basp-m2022.vercel.app/tasks
Test Results: Failed ⚠️

Bugs found:

- Task can't be created with the 'Add' button because it redirects to the localhost (3000), instead of the api.

- The delete function can't be tested due to not having seed data on 'Tasks':

tasks-list

File: src/Components/Tasks/List/Btn/button.module.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+button{
+    height: 100px;
+    background-color: blue;
+}
```
Comment (FefeTroanes) [inline src/Components/Tasks/List/Btn/button.module.css:4]: Che martin creo que aca pasa lo mismo que me dijiste de que github pide que dejes una linea en blanco a lo ultlimo

File: src/Components/Tasks/List/list.module.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+}
+.row {
+    background-color: rgb(243, 240, 240);
+    border-color: rgb(98, 98, 98);
+    border-style: groove;
+    border-width: 3px;
+    border-radius: 10px;
+    padding: 10px;
+
+}
```
Comment (FefeTroanes) [inline src/Components/Tasks/List/list.module.css:21]: Aca tambien

File: src/Components/Tasks/tasks.module.css [hunk 6]
```diff
@@ -1,6 +1,7 @@
 .container {
   flex: 1;
   display: flex;
-  justify-content: center;
+  flex-direction: column;
   align-items: center;
+  padding: 1%;
 }
```
Comment (FefeTroanes) [inline src/Components/Tasks/tasks.module.css:7]: Y aca lo mismo

**Comment by RominaRIP** [comment]
## 2nd Testing results

Tested and working well:
- Table list
- Delete/Edit icon
- Delete feature
- Redirect to Add form
- Warning message for delete

The functionality for 'Redirect to Add form' redirects to 'Home', same happens when when you click on the 'Edit' button, but we understand that it will be fixed when you can merge this PR.

Everything looks good, Martín. Nice job!

**Pull request merged**